libsolidity/interface/OptimiserSettings.h

```diff
@@ -44,7 +44,7 @@ struct OptimiserSettings
 	static char constexpr DefaultYulOptimiserSteps[] =
 		"dhfoDgvulfnTUtnIf"            // None of these can make stack problems worse
 		"["
-			"xarrscLM"                 // Turn into SSA and simplify
+			"xarrsxuscLM"              // Turn into SSA and simplify
```

Comment:
Why is it `sc` and not `cs` by the way? `c` is common subexpression eliminator and `s` is expression simplifier and the expression simplifier docs say it is better if `c` is run beforehand.

Reply:
I checked to make sure I did not introduce this by mistake when refactoring the optimizer suite to use a sequence but looks like it was an intentional tuning that you did a long time ago in #5864 to fix #5773 :)

> Why isn't `add(headStart44, value4)` simplified to `value4`?
> Especially since we already use the fact that `value4` is zero in the revert call above...

> Unfortunately, I cannot really write a test, but this change eliminated an "add 0" in another (complicated) setup.